.idea/compiler.xml

```diff
@@ -2,8 +2,14 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <bytecodeTargetLevel>
+      <module name="content_script_main" target="1.8" />
```

Comment:
Is it on purpose that parts of `.idea/` are not in gitignore?

Reply:
Yes, some of the files in `.idea` are useful to check in. See https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore